.eslintrc.yml

```diff
@@ -0,0 +1,17 @@
+env:
+  node: true
+
+extends:
+  - eslint:recommended
+
+root: true
+
+rules:
+  eol-last: error
+  eqeqeq: error
+  handle-callback-err: error
+  indent: [error, 2]
+  no-unused-vars: [error, {vars: all, args: none}]
+  one-var: [error, never]
+  quotes: [error, single]
+  semi: [error, always]
```

Comment:
Once we're happy with the baseline ESLint config, we probably want to make our own shared `eslint-config-shield-study` config that we can share between __mozilla/shield-study-*__ repos.